ddl/multi_schema_change.go

```diff
@@ -106,12 +282,15 @@ func checkOperateSameColAndIdx(info *model.MultiSchemaInfo) error {
 	if err := checkColumns(info.DropColumns, true); err != nil {
 		return err
 	}
-	if err := checkColumns(info.RelativeColumns, false); err != nil {
+	if err := checkColumns(info.PositionColumns, false); err != nil {
 		return err
 	}
 	if err := checkColumns(info.ModifyColumns, true); err != nil {
 		return err
 	}
+	if err := checkColumns(info.RelativeColumns, false); err != nil {
```

Comment:
seems we can just add `checkColumns(info.PositionColumns, false)` here, no need to change L109

Reply:
Sorry, could you explain it in detail?

```
L109: subJobs[i] = *sub
```

Reply:
sorry, L285

Reply:
Hmm..

- `PositionColumns` means the column in the position clause, like the "c1" in "alter table t add column c3 int after **c1**".
- `RelativeColumns` means the dependent columns for generated column, like the "c1" and "c2" in "create table t (c1 int, c2 int, c3 int as (**c1** + **c2**))".

Reply:
OK I get your point.

The reason why we put the check for `PositionColumns` **before** `ModifyColumns` is that we would like to support multi-schema change like

```sql
-- create table t (a int, b int, c int);
alter t add column d int after a, modify column a bigint;
```